app-thunderbird/build.gradle.kts

```diff
@@ -0,0 +1,161 @@
+    testImplementation(projects.plugins.openpgpApiLib.openpgpApi)
+    testImplementation(projects.feature.account.setup)
+}
+
+android {
+    namespace = "net.thunderbird.android"
+
+    defaultConfig {
+        applicationId = "net.thunderbird.placeholder"
+        testApplicationId = "net.thunderbird.placeholder.tests"
+
+        versionCode = 1
+        versionName = "0.1-SNAPSHOT"
+
+        // Keep in sync with the resource string array "supported_languages"
+        resourceConfigurations.addAll(
+            listOf(
+                "ar",
+                "be",
+                "bg",
+                "br",
+                "ca",
+                "cs",
+                "cy",
+                "da",
+                "de",
+                "el",
+                "en",
+                "en_GB",
+                "eo",
+                "es",
+                "et",
+                "eu",
+                "fa",
+                "fi",
+                "fr",
+                "fy",
+                "gd",
+                "gl",
+                "hr",
+                "hu",
+                "in",
+                "is",
+                "it",
+                "iw",
+                "ja",
+                "lt",
+                "lv",
+                "ml",
+                "nb",
+                "nl",
+                "pl",
+                "pt_BR",
+                "pt_PT",
+                "ro",
+                "ru",
+                "sk",
+                "sl",
+                "sq",
+                "sr",
+                "sv",
+                "tr",
+                "uk",
+                "zh_CN",
+                "zh_TW",
+            ),
+        )
+
+        buildConfigField("String", "CLIENT_ID_APP_NAME", "\"Thunderbird\"")
+    }
+
+    signingConfigs {
+        if (project.hasProperty("thunderbird.keyAlias") &&
+            project.hasProperty("thunderbird.keyPassword") &&
+            project.hasProperty("thunderbird.storeFile") &&
+            project.hasProperty("thunderbird.storePassword")
+        ) {
+            create("release") {
+                keyAlias = project.property("thunderbird.keyAlias") as String
+                keyPassword = project.property("thunderbird.keyPassword") as String
+                storeFile = file(project.property("thunderbird.storeFile") as String)
+                storePassword = project.property("thunderbird.storePassword") as String
+            }
+        }
+    }
+
+    buildTypes {
+        debug {
+            applicationIdSuffix = ".debug"
+            isMinifyEnabled = false
+            manifestPlaceholders["appAuthRedirectScheme"] = "net.thunderbird.placeholder.debug"
+        }
+
+        release {
+            signingConfig = signingConfigs.findByName("release")
+            isMinifyEnabled = true
+            isShrinkResources = true
+            manifestPlaceholders["appAuthRedirectScheme"] = "net.thunderbird.placeholder"
```

Comment:
Using the same redirect scheme for multiple applications is not recommended. This setup can lead to complications as all registered apps with identical schemes will recognize and process the redirect. In particular, when you have both K-9 Mail and Thunderbird for Android (or their variants like debug, daily, beta, production) installed on your device and one of them is set as the default handler. It will prevent another application from successfully signing in since its response would be handled incorrectly by the wrong app.

Reply:
My bad. #7759 hasn't been merged into `main` yet. But with that change applied, the application ID will be used for the scheme of OAuth redirect URI. That is identical in behavior to what is configured here manually.

Reply:
I'll do this in a follow up pull-request.